### DISCUSSION TITLE
Reports migration from MWS API

### DISCUSSION THREAD

**Discussion by pashamesh** [amzn/selling-partner-api-models#3290] [General]
Good day!
Is there any replacement for MWS API `_GET_ORDERS_DATA_ ` and `GET_FLAT_FILE_ORDER_REPORT_DATA_INVOICING` reports in SP-API?
Is there any information about SP-API reports content structure (columns/attributes)?

**Comment by stefnats** [comment]
Regarding `_GET_ORDERS_DATA_`, I can't tell you if/where this report type in sp Api is.

However, reports should look exactly the same as in MWS, the only difference is the starting and ending underscores.

**Comment by lashaLobjanidzeXentral** [comment]
There is a possible replacement as an aggregation, but I have a different issue. When I try to get one of this reports types in aggregation, I always get the  `"processingStatus": "CANCELLED"`, even though there are data for this report type.

```
'_GET_ORDERS_DATA_' => [
    'GET_ORDER_REPORT_DATA_INVOICING',
    'GET_ORDER_REPORT_DATA_TAX',
    'GET_ORDER_REPORT_DATA_SHIPPING',
],
```
